solr/core/src/java/org/apache/solr/handler/RequestHandlerBase.java

```diff
@@ -131,6 +134,11 @@ public void init(NamedList<?> args) {
     if (initArgs != null) {
       Object caching = initArgs.get("httpCaching");
       httpCaching = caching != null ? Boolean.parseBoolean(caching.toString()) : true;
+      Boolean aggregateNodeLevelMetricsEnabled =
```

Comment:
Assuming this is fairly minor, I left this as is since we are already using it so it'll save us a bit of work from having to switch it. If this is a blocker, let me know and I can change it.